tests/playwright/sanity/atomic-widgets.test.ts

```diff
@@ -47,5 +48,12 @@ test.describe( 'Atomic Widgets', () => {
 			} );
 		} );
 	} );
+
+	test( 'Widgets are displayed in front end', async () => {
+		await editor.publishAndViewPage();
+		await editor.page.setViewportSize( { width: 1920, height: 1080 } );
+		await expect.soft( editor.page.locator( '.page-content' ) )
+			.toHaveScreenshot( 'atomic-widgets-frontend.png' );
+	} );
```

Comment:
Don't you think the test above covers this case?

Reply:
No-this checks that the front end renders well for all widgets and that everything still looks the same